root/static/scripts/edit/URLCleanup.js

```diff
@@ -1747,6 +1747,22 @@ const CLEANUPS = {
       return /^https:\/\/myspace\.com\//.test(url);
     },
   },
+  'napster': {
+    match: [new RegExp('^(https?://)?[a-z]{2}\\.napster\\.com', 'i')],
```

Comment:
It should probably capture both `napster.com` and `www.napster.com` too.

Reply:
I tried, and both https://napster.com/artist/bread and https://www.napster.com/artist/bread go to broken pages ("we're not available where you are" kind of stuff), so we don't want to autoselect those. Maybe we should catch and then validate so that we specifically block them though, dunno.

Reply:
These pages redirect to https://fr.napster.com/artist/bread for me.

Reply:
It probably makes sense to enforce `us` subdomain for these just like we do (did?) for iTunes.

Reply:
Hmm, ok, I guess they redirect to your local one then, and if there's no local one (like here) they just tell you it's not available.

Reply:
Added the change to enforce `us.` if we have no subdomain then.